RELEASE-NOTES.txt

```diff
@@ -1,17 +1,20 @@
+4.24
+-----
+
 4.23
 -----
- 
+-   Updated Tags limit to up to 256 characters
+
 4.22
 -----
 -   Fixed several layout issues that affected the Editor
 -   Now it's possible to interactively drag the Scroll Bar within the Editor
--   Updated Tags limit to up to 256 characters
```

Comment:
The "updated tags limit to up to 256 characters" was part of #824.

I'm guessing that when that PR was merged, version 4.22 had already been put in code freeze, but the change had not been merged into develop yet. That would have resulted in no entry for version 4.23 being available in the release notes file, so the change from that PR ended up as part of 4.22.